sig-cli/CONTRIBUTING.md

```diff
@@ -288,6 +267,9 @@ Implementation PRs should
 - mention `@kubernetes/sig-cli-pr-reviews`,
 - __include tests__.
 
+
+//Is this the way it should be? 
```

Comment:
No, I don't think the size of the feature really dictates whether you need e2e tests or not.

When in doubt, add tests.

My best understanding:
* Everything needs unit tests.
* Things that affect the behavior of the CLI input or ouput should have integration tests.
* e2e tests are needed for features that should be tested against a variety of different cluster configurations, or need to have complex assertions of conditions that are not easy to check using a shell script.

I don't know how much in depth we want to go in this section other than just say "you need to add tests for everything you do", and maybe that is good enough for this document.